lib/Language/Haskell/Stylish.hs

```diff
@@ -103,3 +109,37 @@ format :: Maybe ConfigPath -> Maybe FilePath -> String -> IO (Either String Line
 format maybeConfigPath maybeFilePath contents = do
   conf <- loadConfig (makeVerbose True) (fmap unConfigPath maybeConfigPath)
   pure $ runSteps (configLanguageExtensions conf) maybeFilePath (configSteps conf) $ lines contents
+
+
+--------------------------------------------------------------------------------
+-- | Searches Haskell source files in any given folder recursively.
+findFiles :: Bool -> [FilePath] -> IO [FilePath]
+findFiles v fs = mapM (findFilesR v . Just) fs >>= return . concat
```

Comment:
Let's rename this to `findHaskellFiles`?

Reply:
Sure.